internal/compiler/wasm/opa/callgraph.csv

```diff
@@ -0,0 +1,2325 @@
+opa_agg_count,opa_value_type
```

Comment:
This is generated from a graphviz dot file. We could also commit that one... it could be interesting to see changes to this file when runtime code is changed.